frontend/src/util/PWValueCheck.tsx

```diff
@@ -0,0 +1,54 @@
+import { Divider } from "primereact/divider";
+import {
+  containsLowercase,
+  containsNumber,
+  containsUppercase,
+} from "./PWCheck";
+
+const PASSWORD_LENGTH = 8;
+
+export const pwCheckHeader = (
+  <div className="font-bold mb-3">Pick a password</div>
+);
+export const pwCheckFooter = (
+  <>
+    <Divider />
+    <p className="mt-2">Suggestions</p>
+    <ul className="pl-2 ml-2 mt-0 line-height-3">
+      <li>At least one lowercase</li>
+      <li>At least one uppercase</li>
+      <li>At least one numeric</li>
+      <li>Minimum 8 characters</li>
+    </ul>
+  </>
+);
+
+export function passwordValueCheck(
```

Comment:
In the future, we should try not to do these messy if statements. It's fine for this time since it's isolated, but there's usually a cleaner way to avoid nesting 5 if statements. You can just invert what you're checking to have 5 sequential if statements, rather than nesting them

if (password1 != password2) {
   return false, "..."
}

if (!(password1.length >= PASSWORD_LENGTH && password2.length >= PASSWORD_LENGTH))
    return false, "..."
}
etc